lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/MavenLemminxExtension.java

```diff
@@ -155,6 +158,8 @@ public void start(InitializeParams params, XMLExtensionsRegistry registry) {
 			registry.registerDiagnosticsParticipant(diagnosticParticipant);
 			hoverParticipant = new MavenHoverParticipant(this);
 			registry.registerHoverParticipant(hoverParticipant);
+			codeActionParticipant = new MavenCodeActionParticipant();
+			registry.registerCodeActionParticipant(codeActionParticipant);

```

Comment:
As mentioned on the other patch, it should be preferred to register each participant individually instead of grouping them.

Reply:
Yes. this is subject to change